brave/brave-core
# Don't allow users to submit Android webcompat reports from browser error pages

Resolves https://github.com/brave/brave-browser/issues/36484



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Go to https://badssl.com/ and go to any of the SSL error pages like `expired`
2. Open Brave Shields pop up dialog.
3. Switch off Brave Shields if It is ON.
4. Click "Report broken site".
5. Make sure that you can;t send report for error page
